src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -165,29 +165,29 @@ uct_rc_mlx5_iface_poll_rx_cq(uct_rc_mlx5_iface_common_t *mlx5_common_iface,
 }
 
 static UCS_F_ALWAYS_INLINE void*
-uct_rc_mlx5_iface_common_data(uct_rc_mlx5_iface_common_t *mlx5_iface,
+uct_rc_mlx5_iface_common_data(uct_rc_mlx5_iface_common_t *iface,
                               uct_rc_iface_t *rc_iface, struct mlx5_cqe64 *cqe,
                               unsigned byte_len, unsigned *flags)
 {
     uct_ib_mlx5_srq_seg_t *seg;
     uct_ib_iface_recv_desc_t *desc;
     void *hdr;
 
-    seg  = uct_ib_mlx5_srq_get_wqe(&mlx5_iface->rx.srq, ntohs(cqe->wqe_counter));
+    seg  = uct_ib_mlx5_srq_get_wqe(&iface->rx.srq, ntohs(cqe->wqe_counter));
     desc = seg->srq.desc;
 
     /* Get a pointer to AM or Tag header (after which comes the payload)
      * Support cases of inline scatter by pointing directly to CQE.
      */
     if (cqe->op_own & MLX5_INLINE_SCATTER_32) {
         hdr = (uct_rc_hdr_t*)(cqe);
-        uct_rc_mlx5_iface_common_rx_inline(mlx5_iface, rc_iface, desc,
+        uct_rc_mlx5_iface_common_rx_inline(iface, rc_iface, desc,
```

Comment:
now, no need to pass `rc_iface` as a separate parameter, because it can be obtained from `iface`. Plz, check *all* functions which used `mlx5_common` and `rc_iface`

Reply:
Now I see that most of funcs addressed in the next commit